llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -954,9 +986,10 @@ int main(int argc, char **argv) {
   bool DoSpecConst = SpecConstLower.getNumOccurrences() > 0;
   bool DoParamInfo = EmitKernelParamInfo.getNumOccurrences() > 0;
   bool DoProgMetadata = EmitProgramMetadata.getNumOccurrences() > 0;
+  bool DoExportedSyms = EmitExportedSymbols.getNumOccurrences() > 0;
```

Comment:
I guess this is produced by a copy-pasting as well as almost all boolean opts above, but strictly speaking, you can pass `false`/`0` to the option, which would mean that number of occurrences would be one, but corresponding `EmitExportedSymbols` flag would be evaluated as `false`, making the message emitted below on 1022 incorrect.

This is definitely a minor comment - we can leave it as-is, because the tool is not used directly by end-users anyway; or perform a refactoring in a separate PR for all boolean opts all together